File: apps/zxtune-android/src/app/zxtune/fs/zxart/Track.java
/**
 *
 * @file
 *
 * @brief Track description POJO
 *
 * @author vitamin.caig@gmail.com
 *
 */

package app.zxtune.fs.zxart;

public final class Track {

  public final int id;
  public final String filename;
  public final String title;
  public final String votes;
  public final String duration;
  public final int year;
  public final String compo;
  public final int partyplace;
  
  public Track(int id, String filename, String title, String votes, String duration, int year, String compo, int partyplace) {
    this.id = id;
    this.filename = filename;
    this.title = title;
    this.votes = votes;
    this.duration = duration;
    this.year = year;
    this.compo = compo;
    this.partyplace = partyplace;
  }
}

File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/fs/zxart/Track.kt
/**
 * @file
 * @brief Track description POJO
 * @author vitamin.caig@gmail.com
 */
package app.zxtune.fs.zxart

data class Track(
    val id: Int,
    val filename: String,
    val title: String = "",
    val votes: String = "",
    val duration: String = "",
    val year: Int = 0,
    val compo: String = "",
    val partyplace: Int = 0
)
